cypress/pages/workflow.ts

```diff
@@ -440,5 +443,8 @@ export class WorkflowPage extends BasePage {
 				top: +$el[0].style.top.replace('px', ''),
 			}));
 		},
+		clearNotifications: () => {
+			this.getters.successToast().find('.el-notification__closeBtn').click({ multiple: true });
+		},
```

Comment:
nit: Should this be somewhere else than on the workflow page's actions, as notifications are shown everywhere?